### PR TITLE
Add links to additional resources to issue templates

### PR DESCRIPTION
**Pull request by soujay**
Added a config.yml file that adds links to ParitcularDiscuss where users can ask a question, and other support options when opening an issue.

image

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    about: Please contact us to discuss your support needs.
+  - name: Particular Community Support
+    url: https://discuss.particular.net/
+    about: Reach out to the particular discourse community.
+  - name: Particular consulting
+    url: https://particular.net/consulting
+    about: Looking for a quick assessment of your major architectural choices before starting a large development effort.
+  - name: Particular onsite tranining
+    url: https://particular.net/onsite-training
+    about: Please contact us for any onsite training.
```
Comment (awright18) [inline .github/ISSUE_TEMPLATE/config.yml:15]:
```suggestion
  - name:  Ask a question
    url: https://discuss.particular.net/
    about: Reach out to the particular discourse community.
  - name: Get support
    url: https://particular.net/support
    about: Please contact us to discuss your support needs.

```

Comment (awright18) [reply .github/ISSUE_TEMPLATE/config.yml:15]: Although it's clever, in my opinion, consulting/training is too much for a GitHub repo. Also I'm suggesting ordering the options in lighter weight (Ask a question) to heavier weight  (Get support). I'm open to asking for outside opinions. I do like that this lets us link to other places.

**Comment by adamralph** [comment]
Could we have a better PR title?

Note that the final commit message (assuming the commits will be squashed) should be meaningful too.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Ask a question
+    url: https://discuss.particular.net/
+    about: Reach out to the particular discourse community.
```
Comment (adamralph) [inline .github/ISSUE_TEMPLATE/config.yml:5]:
```suggestion
    about: Reach out to the ParticularDiscussion community.
```

image

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+  - name: Get support
+    url: https://particular.net/support
+    about: Please contact us to discuss your support needs.
```
Comment (adamralph) [inline .github/ISSUE_TEMPLATE/config.yml:8]:
```suggestion
    about: Contact us to discuss your support requirements.
```

**Review by adamralph — COMMENTED** [review]
Given that https://particular.net/support lists all options, including the discussion forum, do we need more than one link which points to that page?

**Comment by awright18** [comment]
> Could we have a better PR title?

Updated the title

**Comment by awright18** [comment]
> Given that https://particular.net/support lists all options, including the discussion forum, do we need more than one link which points to that page?

@adamralph It's a good question, at the very least I think a link to ParticularDiscuss should be there, the Get support I struggle with because I'm not sure if that link would seem out of place in this context or not. We had several others that we removed because they were marketing-related. You could probably make the case for "Get Support" as well, but on the other hand, it attempts to convey that we do offer paid support if someone needs it.

**Review by adamralph — APPROVED** [review]

**Review by DavidBoike — APPROVED** [review]

**Pull request merged**